packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss

```diff
@@ -51,6 +51,24 @@
         }
     }
 
+    &:hover {
+         &:not(.CheckoutDeliveryOption_isHoverExcluded){
```

Comment:
@AleksandrsKondratjevs  what do you think ???

Reply:
Won't it go with just `&:not(&_isHoverExcluded)` ?

Reply:
Hey, tried and didn't work